scylladb/sphinx-scylladb-theme
# Hide copybutton class

Closes #304

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/examples/code-blocks/#hide-copy-button with your favorite browser. This code block must not have a copy button on hover:

![image](https://user-images.githubusercontent.com/9107969/149139387-4f3d94aa-4767-4c78-b991-939f4c326037.png)
